src/components/TouristInfoCards.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import CreateCards from "./CreateCards";
+
+const Information = [
+  {
+    link: "https://peoplemakeglasgow.com/",
+    image:
```

Comment:
the `Information` array contains a lot of static data that will not change, so I think the component would be cleaner if this data was moved to a json file and just imported here.